### PR TITLE
{bio}[foss/2021b] NGSpeciesID v0.1.2.1 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires:
* #16107

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3146.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/efa250d5ae0953dd0551267eed1cb0a2 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/61c1f226f382787415e332f1c674f83c for a full test report.

File: easybuild/easyconfigs/n/NGSpeciesID/NGSpeciesID-0.1.2.1-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    ('medaka', '1.6.0'),
+    ('spoa', '4.0.7'),
+    ('Racon', '1.5.0'),
+    ('minimap2', '2.22'),
+]
+
+download_dep_fail = True
+use_pip = True
+
+# strip away too strict version requirement for parasail and
```
Comment (verdurin) [inline easybuild/easyconfigs/n/NGSpeciesID/NGSpeciesID-0.1.2.1-foss-2021b.eb:27]: Bit nitpicky, but is there a typo in the comment here?

Comment (boegel) [reply easybuild/easyconfigs/n/NGSpeciesID/NGSpeciesID-0.1.2.1-foss-2021b.eb]: The stripping isn't needed at all actually for this version, so I've removed it in d7478421df

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3105.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/4d9699e70eaab21a140a495a644b95a8 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/4bc2c9f86e59ffeb202501e668a6bdbb for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**